UltiSnips/gitcommit.snippets

```diff
@@ -0,0 +1,95 @@
+snippet chore "create CHORE: commit message"
+CHORE: ${1:add build script}
```

Comment:
but it provides the correct format that if used by everyone... it'll generate a unified commit message across the whole team/projects